src/compareVersions.js

```diff
@@ -0,0 +1,22 @@
+/* exported jQueryVersionSince, compareVersions */
+
+// Returns 0 if v1 == v2, -1 if v1 < v2, 1 if v1 > v2
+function compareVersions( v1, v2 ) {
+	var rVersionParts = /^(\d+)\.(\d+)\.(\d+)/,
```

Comment:
I'm relatively new to game of "optimise for delivery size", but I'll try to jump in here based on what I see us do elsewhere.

I wonder if it'd make sense here to omit the variable in favour of inlining the regex literal in the two places we need it. It would probably be the same or maybe slightly bigger after minification/obfuscation, but after gzip I'd expect it to be slightly smaller that way. Execution speed would presumably be the same or better same. Historically I'd think "parse twice" is slower than "parse once, store, reference twice". But given how regex literals work in JS, I think it might even run slightly faster without the variables.

Anyhow, super low priority :) I'm just testing out what our reduction thoughts are. I imagine it's generally more important in core than in migrate. (Although Wikipedia and WordPress do both run Migrate in production for prolonged periods of time, given the need to support user-supplied scripts.)